### DISCUSSION TITLE
Does Cog support changing WebKitSettings options?

### DISCUSSION THREAD

**Discussion by bkardell** [Igalia/cog#229] [Q&A] [answered]
it seems the ability to set these options is not exposed to WPE, so it is for example, impossible to redirect console errors to std-err, which makes things considerably difficult.

**Comment by bkardell** [comment]
Disregard - irrelevant, I guess I misunderstood where the problem was

**Comment by tobias-bischoff** [comment, marked as answer]
Reference for future readers: https://github.com/Igalia/cog/issues/27 – run cog with `—help-websettings`
